app/helpers/components_helper.rb

```diff
@@ -9,9 +10,23 @@ def strategy_path(country)
     end
   end
 
+  def define_path(page)
+    return send(to_path(page[:slug]), locale: 'en') if static_page?(page)
```

Comment:
Just a spot. I hope that we don't have a request for Welsh on Fincap. Because the locale is hardcoded in a lot of places. It will be good for we start to use the default_url_options in a near future. Like frontend: https://github.com/moneyadviceservice/frontend/blob/master/app/controllers/concerns/localisation.rb#L21
https://github.com/moneyadviceservice/frontend/blob/master/app/controllers/application_controller.rb#L14
cc fyi @your 
It is not a blocker for this PR but it is worth to mention.

Reply:
@tomas-stefano yeah, the hardcoded locale is probably going to be a problem. Would be nice to address it in a separate ticket, thanks for raising it.

Reply:
@tomas-stefano agree, please would you raise a ticket

Reply:
I am unsure if I should. Ryan always says to not raise a tech debt issue and fix right away. I am unsure if I should. Also there is no ticket for have welsh on fincap atm.